launch/launch/exit_handler.py

```diff
@@ -33,7 +33,8 @@ def default_exit_handler(context):
         context.launch_state.teardown = True
 
     # set launch return code if not already set
-    if not context.launch_state.returncode:
+    # TODO remove second condition, currently fails on Windows
```

Comment:
Could you add your username to the `TODO`? i.e. `TODO(dirk-thomas)`, makes it easier to know who wrote this code in the future.